eth/tracers/tracer.go

```diff
@@ -662,6 +662,12 @@ func (jst *Tracer) GetResult() (json.RawMessage, error) {
 		case *big.Int:
 			pushBigInt(val, jst.vm)
 
+		case int:
+			jst.vm.PushInt(val)
+
+		case common.Hash:
+			jst.vm.PushString(val.Hex())
```

Comment:
Hmm, wondering if this should be similar to the `common.Address` part:

```go
			ptr := jst.vm.PushFixedBuffer(32)
			copy(makeSlice(ptr, 32), val[:])
```

Reply:
Done!